cmd/upload.go

```diff
@@ -110,7 +110,7 @@ var uploadCmd = &cobra.Command{
 			return err
 		}
 
-		if err = ioutil.WriteFile("report.json", data, 0644); err != nil {
+		if err = ioutil.WriteFile("report.json", data, 0o644); err != nil {
```

Comment:
This is an error. The o should not be added.

Reply:
I still want to understand how this thing works...

according to https://pkg.go.dev/io/ioutil#WriteFile this func calls `os.WriteFile` internally.
Preflight code calls `os.WriteFile` directly in its code, and the `0o` which gofumpt added, seems to work...
but .... there's no mention of `o` in https://pkg.go.dev/os#FileMode ?

Reply:
meh Octal....  ok I guess resolved then @mrhillsman ?